tests/integration/test_logcollector/test_configuration/test_basic_configuration_reconnect_time.py

```diff
@@ -0,0 +1,142 @@
+'''
+copyright: Copyright (C) 2015-2022 Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The 'wazuh-logcollector' daemon monitors configured files and commands for new log messages.
+       Specifically, these tests will check if the logcollector detects invalid values for the
+       'reconnect_time' tag. Log data collection is the real-time process of making sense out
+       of the records generated by servers or devices. This component can receive logs through
+       text files or Windows event logs. It can also directly receive logs via remote syslog
+       which is useful for firewalls and other such devices.
+
+components:
+    - logcollector
+
+suite: configuration
+
+targets:
+    - agent
+
+daemons:
+    - wazuh-logcollector
+
+os_platform:
+    - windows
+
+os_version:
+    - Windows 10
+    - Windows 8
+    - Windows 7
+    - Windows Server 2019
+    - Windows Server 2016
+    - Windows Server 2012
+    - Windows Server 2003
+    - Windows XP
+
+references:
+    - https://documentation.wazuh.com/current/user-manual/capabilities/log-data-collection/index.html
+    - https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html#reconnect-time
+
+tags:
+    - logcollector_configuration
+'''
+import pytest
+import sys
+
+from pathlib import Path
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.constants.platforms import WINDOWS
+from wazuh_testing.constants.daemons import LOGCOLLECTOR_DAEMON
+from wazuh_testing.modules.logcollector import patterns
+from wazuh_testing.tools.monitors import file_monitor
+from wazuh_testing.utils import callbacks, configuration
+from wazuh_testing.utils.services import control_service
+from wazuh_testing.utils.file import truncate_file
+
+from . import TEST_CASES_PATH, CONFIGURATIONS_PATH
+
+# Marks
+if sys.platform == WINDOWS:
+    pytestmark = pytest.mark.tier(level=0)
+else:
+    pytestmark = [pytest.mark.skip, pytest.mark.tier(level=0)]
```

Comment:
Reason?

Reply:
The [reconnect_time](https://documentation.wazuh.com/current/user-manual/reference/ossec-conf/localfile.html#reconnect-time) tag only applies when the `log_format `tag is `eventchannel`, `eventchannel `only works on windows.